packages/kcmsf/src/pages/result.tsx

```diff
@@ -0,0 +1,250 @@
+import { Flex, Select, Table, Title } from "@mantine/core";
+import { DepartmentType, config } from "config";
+import { useEffect, useState } from "react";
+
+type PreMatch = {
+  id: string;
+  matchCode: string;
+  departmentType: DepartmentType;
+  leftTeam: {
+    id: string;
+    teamName: string;
+  };
+  rightTeam: {
+    id: string;
+    teamName: string;
+  };
+  runResults: [
+    {
+      id: string;
+      teamID: string;
+      points: number;
+      goalTimeSeconds: number | null;
+      finishState: "goal" | "finished";
+    },
+  ];
```

Comment:
これだと"`{id: string; ... }`が1個入ったタプル型"です。配列は
```suggestion
  runResults: {
      id: string;
      teamID: string;
      points: number;
      goalTimeSeconds: number | null;
      finishState: "goal" | "finished";
    }[],
```
のようにします。L41~も同様です。